### ISSUE TITLE
Tidy up imports - remove all `from x import *`

### ISSUE THREAD

**Issue by nbarlowATI** [alan-turing-institute/AIrsenal#276]
For historic reasons / speed of development we have several places where we do 
```
from <some-module> import *
```

However, this is not good practice, and at this point it would be worth specifying exactly what we're importing. i.e. 
```
from <some-module> import (
  <something>,
  <something-else>,
  ...
)
```

It might involve trial and error trying to track these down, i.e. remove the ` ... import *` line, try to run the code, and see what complains.  For example, if we remove the line `from ..framework.utils import *` from `airsenal/scripts/update_results_transactions_db.py` then run `airsenal_update_db` we get:
```
NameError: name 'CURRENT_SEASON' is not defined
```
because we need to import `CURRENT_SEASON` from utils.
 
Given that `pip install .` can be quite slow, this trial-and-error might be a bit time-consuming, so maybe we won't do it all in one go, but every step towards eliminating `import *` will be helpful! :)

**Comment by anguswilliams91** [comment]
This may be easier to do in an IDE, like pycharm (they'll often underline functions that aren't imported. Pycharm might even have some magic to "tidy up" `import *`, but im not sure about that)

**Comment by Abelarm** [comment]
I can take care of this, I'll try to clean all the import * in one PR if it's too long I will split in different PRs

**Comment by nbarlowATI** [comment]
Great, many thanks @Abelarm ! :)

**Comment by Abelarm** [comment]
I have create the first PR, I removed the all the `from x import *` and also removed unused imports. 
I think I also found a bug since there was one call to a function called `fill_session_team` but no definition of this function in the repo, the only function which had the same signature was `fill_session_squad` so I thought about a typo or something so i changed it.

**Comment by nbarlowATI** [comment]
Closed by #284

**Comment by Abelarm** [comment]
I think it miss one PR, I didn't complete the fixing of the test directory yet... tomorrow you'll have it
